packages/@aws-cdk/aws-lambda-nodejs/lib/types.ts

```diff
@@ -66,6 +66,65 @@ export interface BundlingOptions {
    */
   readonly keepNames?: boolean;
 
+  /**
+   * Normally the esbuild automatically discovers `tsconfig.json` files and reads their contents during a build.
+   *
+   * However, you can also configure a custom `tsconfig.json` file to use instead.
+   *
+   * This can be useful if you need to do multiple builds of the same code with different settings:
```

Comment:
Mention this is relative to the input dir

Reply:
I don't like the fact that it's relative to the input dir, the user should be able to point to a path and it's up to the construct to provide it correctly to `esbuild`.

This is similar to what is done with `entry` and its relative path calculated here:
https://github.com/aws/aws-cdk/blob/8f6f9a8678496e131a43ca4c76e561d50a0a0de8/packages/%40aws-cdk/aws-lambda-nodejs/lib/bundling.ts#L64-L65

Reply:
Make sense!

Reply:
Makes more sense, Thanks for highlighting!